latest/docs-ref-conceptual/tutorial-azure-cli-server.yml

```diff
@@ -0,0 +1,246 @@
+    which can be used in-browser as part of this tutorial. Or you can [install the Azure CLI](install-azure-cli.md)
+    and follow along in your own terminal.
+
+- title: Create a Virtual Machine
+  durationInMinutes: 4
+  content: |
+      First, log in with your Azure subscription information. Follow any on-screen commands to get
+      your shell configured and authenticate with Azure.
+
+      ```azurecli
+      az login
+      ```
+
+      The first step to creating resources with Azure is to have a resource group which will
+      contain and associate them. For this tutorial, we will want a dedicated resource group
+      in a region which has availability for all resources you will create. You do this with
+      the [az group create](/cli/azure/group#az_group_create) command.
+
+      ```azurecli
+      az group create --name TutorialResources --location westeurope
+      ```
+
+      You can now create a VM and its associated resources in this resource group with [az vm create](/en-us/cli/azure/vm#az_vm_create).
+
+      ```azurecli
+      az vm create --resource-group TutorialResources --name TutorialVM --image UbuntuLTS --generate-ssh-keys
+      ```
+
+      This creates a VM for you with the standard configuration and security policies. Behind the scenes it also creates all of the required resources
+      that a VM needs: A storage account, a storage disk to hold the OS image, a virtual network (VNET), a subnet, a virtual network interface (NIC),
+      a reserved public IP address, and a network security group (NSG). Later in this tutorial you will work with some of these resources. This also
+      generates a new `.ssh/id_rsa` SSH key, for logging into the VM.
+
+      > [!NOTE]
+      > If your system already has an `.ssh/id_rsa` key, a new key will not be generated and this key will be used instead.
+
+      You should see output similar to the following.
+
+      ```output
+      {
+          "fqdns": "",
+          "id": "/subscriptions/0b186e08-ea4c-4dcf-a82e-da80567adc1c/resourceGroups/TutorialResources/providers/Microsoft.Compute/virtualMachines/TutorialVM",
+          "location": "westeurope",
+          "macAddress": "00-0D-3A-20-63-23",
+          "powerState": "VM running",
+          "privateIpAddress": "10.0.0.4",
+          "publicIpAddress": "52.232.70.227",
+          "resourceGroup": "TutorialResources",
+          "zones": ""
+      }
+      ```
+
+      The output gives you some information about the network and power state of the newly created VM. At this point you should record the value of `publicIpAddress` so that you can
+      connect to the VM and confirm that it is running and authenticates correctly over SSH.
+
+      ```bash
+      ssh <IP ADDRESS>
+      ```
+
+      If everything is working correctly, you should see Ubuntu's welcome message.  Log out, and now you're ready for the next step of creating and attaching a data disk.
+
+- title: Create and attach a data disk
```

Comment:
What's the point of having the user attach a disk? Seems like it's the most complicated step in this tutorial and we're only doing it for the sake of doing it-not because the web server needs it.

Reply:
I was considering cutting it. The only interesting thing about it is that it demonstrates manually tying Azure resources together. If you think it would be better not being there, I'm fine with removing it.